documentation/api/query/v4/facts.markdown

```diff
@@ -80,7 +80,7 @@ Subquery against `/fact-contents` to get all remotely-authenticated trusted fact
 
     curl -X GET http://localhost:8080/v4/facts --data-urlencode 'query=["in", ["name","certname"],
       ["extract",["name","certname"],
-        ["select-fact-contents", ["~>", "path", [".*", "authenticated"]]]]]'
+        ["select_fact_contents", ["~>", "path", [".*", "authenticated"]]]]]'
```

Comment:
fixed the bit about the export filenames. I don't really like the idea of underscored endpoints but I'll go along with it if that's what we decide we want. You're definitely right about the need to generalize the operator.